src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -100,7 +103,8 @@ boot_newcomp_create(int spdid, struct cos_compinfo *comp_info)
 	assert(main_thd);
 
 	/* Add created component to "scheduling" array */
-	while (schedule[i] != 0) i++;
+	while (schedule[i] != 0)
```

Comment:
Frankly I don't understand what the formatter is doing here. This should be allowed.